<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEditSearch"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxSearch"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tableView">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>clients</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>clientsrequests</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>deals</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>employees</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>immovables</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>immovablestypes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>owners</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ownersrequests</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonQuery">
        <property name="text">
         <string>Query</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEditing">
    <property name="title">
     <string>Editing</string>
    </property>
    <addaction name="actionAdd_row"/>
    <addaction name="actionEdit_row"/>
    <addaction name="actionDelete_row"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_the_author"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEditing"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_row"/>
   <addaction name="actionEdit_row"/>
   <addaction name="actionDelete_row"/>
   <addaction name="separator"/>
   <addaction name="actionAbout_the_author"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/pico/icon/connect.png</normaloff>:/pico/icon/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/pico/icon/close.png</normaloff>:/pico/icon/close.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/pico/icon/exit.png</normaloff>:/pico/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEdit_row">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/pico/icon/edit.png</normaloff>:/pico/icon/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit row</string>
   </property>
  </action>
  <action name="actionAdd_row">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/pico/icon/addNew.png</normaloff>:/pico/icon/addNew.png</iconset>
   </property>
   <property name="text">
    <string>Add row</string>
   </property>
  </action>
  <action name="actionDelete_row">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/pico/icon/delete.png</normaloff>:/pico/icon/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete row</string>
   </property>
  </action>
  <action name="actionAbout_the_author">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/pico/icon/about.png</normaloff>:/pico/icon/about.png</iconset>
   </property>
   <property name="text">
    <string>About the author</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionGermany">
   <property name="text">
    <string>Germany</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
